lib/web_ui/lib/src/engine/text_editing/autofill_hint.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+part of engine;
+
+/// Maps AutofillHints from the framework to the autofill hints that is used for
+/// browsers.
+/// See: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/autofill.dart
+/// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
+class BrowserAutofillHints {
+  static final BrowserAutofillHints _singletonInstance =
```

Comment:
nit: can we make this public and remove the public getter?

Reply:
We usually use this as a singleton pattern in the web engine. for example: https://github.com/flutter/engine/blob/master/lib/web_ui/dev/firefox_installer.dart#L17